pynxtools/dataconverter/writer.py

```diff
@@ -260,6 +266,8 @@ def add_units_key(dataset, path):
                                                          self.output_path, path])
                     else:
                         dataset = grp.create_dataset(entry_name, data=data)
+            except InvalidDictProvided as exc:
+                print(str(exc))
```

Comment:
Wouldn't it be better to re-raise the error here?

Reply:
This we have resolved over slack.